nomad/local/run_integration_tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -euo pipefail
+
+GRAPL_ROOT="$(git rev-parse --show-toplevel)"
+THIS_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+# shellcheck source-path=SCRIPTDIR
+source "${THIS_DIR}/nomad_cli_tools.sh"
+
+echo "--- Deploying integration tests"
+
+# Wait a short period of time before attempting to deploy infrastructure
+# shellcheck disable=SC2016
+timeout 30 bash -c -- 'while [[ -z $(nomad status 2>&1 | grep running) ]]; do printf "Waiting for nomad-agent\n";sleep 1;done'
```

Comment:
Not really needed - seems highly unlikely nomad would not be up already by the time we did this (like, a Pulumi deploy would have happened first)